src/main/java/com/gregtechceu/gtceu/api/recipe/lookup/GTRecipeLookup.java

```diff
@@ -581,6 +607,66 @@ private boolean recurseIngredientTreeAdd(@NotNull GTRecipe recipe,
         return true;
     }
 
+    /**
+     * Removes a recipe from the map. (recursive part)
+     *
+     * @param recipeToRemove the recipe to add.
+     * @param ingredients    list of input ingredients.
+     * @param branchMap      the current branch in the recursion.
+     */
+    private @Nullable GTRecipe recurseIngredientTreeRemove(@NotNull GTRecipe recipeToRemove,
```

Comment:
remove. this won't work. it'd just make recipes that are in the recipe viewer not work.